### ISSUE TITLE
Unexpected Conn-Track Drops

### ISSUE THREAD

**Issue by danwent** [cilium/cilium#2644]
I'm seeing conn-track continue to drop reply packets from a kube-api server running on the host to some of my pods (c-agent, c-exporter) that are talking to the kube-api server.   I got into this state when I had to restart the cilium pod due to another issue.  

I would expect that since there apparently was a legitimate connection in place here that the conn-track would be able to allow this connection to continue, or if not, tear it down.  Right now, these ack packets to the same empheral ports continue to be sent, even 20+ minutes after the cilium restart.   

````
xx drop (CT: Can't create entry from packet), to endpoint 31639, identity 1->288: 10.0.2.15:8443 -> 10.15.49.27:55930 tcp ACK
xx drop (CT: Can't create entry from packet), to endpoint 35132, identity 1->277: 10.0.2.15:8443 -> 10.15.249.57:33576 tcp ACK
xx drop (CT: Can't create entry from packet), to endpoint 31639, identity 1->288: 10.0.2.15:8443 -> 10.15.49.27:55930 tcp ACK
xx drop (CT: Can't create entry from packet), to endpoint 35132, identity 1->277: 10.0.2.15:8443 -> 10.15.249.57:33576 tcp ACK
xx drop (CT: Can't create entry from packet), to endpoint 31639, identity 1->288: 10.0.2.15:8443 -> 10.15.49.27:55930 tcp ACK
^C
Received an interrupt, disconnecting from monitor...

root@minikube:~# 
root@minikube:~# cilium endpoint list
ENDPOINT   POLICY (ingress)   POLICY (egress)   IDENTITY   LABELS (source:key[=value])                      IPv6                 IPv4            STATUS   
           ENFORCEMENT        ENFORCEMENT                                                                                                        
8858       Disabled           Disabled          298        k8s:app=empire-backup                            f00d::a0f:0:0:229a   10.15.104.229   ready   
                                                           k8s:io.kubernetes.pod.namespace=staging-kafka3                                                
12164      Disabled           Disabled          294        k8s:app=zook                                     f00d::a0f:0:0:2f84   10.15.117.21    ready   
                                                           k8s:io.kubernetes.pod.namespace=staging-kafka3                                                
15009      Disabled           Disabled          296        k8s:app=empire-outpost                           f00d::a0f:0:0:3aa1   10.15.218.41    ready   
                                                           k8s:io.kubernetes.pod.namespace=staging-kafka3                                                
29898      Disabled           Disabled          299        reserved:health                                  f00d::a0f:0:0:74ca   10.15.242.54    ready   
31639      Disabled           Disabled          288        k8s:app=c-exporter                        f00d::a0f:0:0:7b97   10.15.49.27     ready   
                                                           k8s:io.kubernetes.pod.namespace=kube-system                                                   
34774      Disabled           Disabled          297        k8s:app=empire-hq                                f00d::a0f:0:0:87d6   10.15.154.57    ready   
                                                           k8s:io.kubernetes.pod.namespace=staging-kafka3                                                
35132      Disabled           Disabled          277        k8s:app=c-agent                           f00d::a0f:0:0:893c   10.15.249.57    ready   
                                                           k8s:io.kubernetes.pod.namespace=kube-system                                                   
50211      Enabled            Disabled          295        k8s:app=kafka                                    f00d::a0f:0:0:c423   10.15.119.87    ready   
                                                           k8s:io.kubernetes.pod.namespace=staging-kafka3                                                
63264      Disabled           Disabled          296        k8s:app=empire-outpost                           f00d::a0f:0:0:f720   10.15.222.0     ready   
                                                           k8s:io.kubernetes.pod.namespace=staging-kafka3  
````

Here is the conn-track table, clearly not showing any entry for them: 

````
root@minikube:~# cilium bpf ct list global                           

ICMP IN 10.15.242.54:0 -> 10.15.28.238:0 related  expires=12758 rx_packets=1 rx_bytes=74 tx_packets=0 tx_bytes=0 flags=0 revnat=0 proxyport=0 src_sec_id=1
TCP IN 10.15.117.21:2181 -> 10.15.119.87:38132  expires=12758 rx_packets=1806 rx_bytes=130511 tx_packets=906 tx_bytes=77865 flags=0 revnat=0 proxyport=0 src_sec_id=295
ICMP IN 10.15.242.54:2048 -> 10.15.28.238:0  expires=12758 rx_packets=32 rx_bytes=1600 tx_packets=32 tx_bytes=1600 flags=0 revnat=0 proxyport=0 src_sec_id=1
TCP OUT 10.0.2.15:8443 -> 10.15.49.27:39570  expires=12759 rx_packets=104 rx_bytes=70750 tx_packets=109 tx_bytes=9465 flags=0 revnat=1 proxyport=0 src_sec_id=288
TCP IN 10.15.242.54:4240 -> 10.15.28.238:54756  expires=12408 rx_packets=5 rx_bytes=474 tx_packets=5 tx_bytes=464 flags=3 revnat=0 proxyport=0 src_sec_id=1
TCP OUT 10.0.2.15:8443 -> 10.15.249.57:44242  expires=12752 rx_packets=54 rx_bytes=44036 tx_packets=55 tx_bytes=5087 flags=0 revnat=1 proxyport=0 src_sec_id=277
TCP OUT 10.15.117.21:2181 -> 10.15.119.87:38132  expires=12758 rx_packets=906 rx_bytes=77865 tx_packets=1806 tx_bytes=130511 flags=0 revnat=23 proxyport=0 src_sec_id=295
TCP OUT 34.216.125.124:443 -> 10.15.49.27:53246  expires=12759 rx_packets=2205 rx_bytes=199556 tx_packets=1615 tx_bytes=288180 flags=0 revnat=0 proxyport=0 src_sec_id=288
````

**Comment by joestringer** [comment]
This should be fixed by #2700.

**Comment by tgraf** [comment]
We have never seen this again since #2700
